### PR TITLE
[Test][Nfs] Add test to verify multi client file access with rootsquash

### PR DESCRIPTION
**Pull request by pranavprakash20**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by pranavprakash20** [comment]
run log: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-H3UFC2/

**Review by MohitBis — DISMISSED** [review]
LGTM

File: tests/nfs/test_nfs_rootsquash_file_access_from_multiple_clients.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                port=port,
+                server=nfs_server_name,
+                export=nfs_export_squash,
+            ):
+                raise OperationFailedError(
+                    f"Failed to mount nfs on {clients[0].hostname}"
+                )
+            log.info("Mount succeeded on client")
+
+        # Create file on squashed dir and check permission
```
Comment (msainiRedhat) [inline tests/nfs/test_nfs_rootsquash_file_access_from_multiple_clients.py:80]: Where are we checking permission?

**Review by msainiRedhat — APPROVED** [review]
LGTM

**Review by MohitBis — APPROVED** [review]
LGTM

**Review by msainiRedhat — APPROVED** [review]
LGTM

**Review by vamahaja — APPROVED** [review]

**Pull request merged**